src/main/java/ithillel/lesson11/ArrayInitializer.java

```diff
@@ -0,0 +1,29 @@
+package ithillel.lesson11;
+
+public class ArrayInitializer {
+    public static void init(double[] arr){
+        assert arr.length != 0;
+
+        double[] arrThread1 = new double[arr.length/2];
+        double[] arrThread2 = new double[arr.length/2 + 1];
+        boolean isEven = arr.length % 2 == 0;
+
+        System.arraycopy(arr, 0, arrThread1, 0, arr.length/2);
+        System.arraycopy(arr, arr.length/2, arrThread2, 0, arr.length/2 + (isEven ? 0:1));
```

Comment:
```suggestion
        System.arraycopy(arr, arr.length/2, arrThread2, 0, arr.length - arr.lenght/2);
```
Трошки складновуто для розуміння з isEven, здається так також має працювати